android/app/src/main/AndroidManifest.xml

```diff
@@ -90,6 +90,29 @@
                 <data android:mimeType="image/*" />
             </intent-filter>
 
+            <intent-filter>
+                <action android:name="android.intent.action.SEND" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="video/*" />
```

Comment:
IMO, This is not necessary. If you added `mimeType="*/*" />` it already supports all file types.